backend/drive/drive.go

```diff
@@ -1078,13 +1083,19 @@ func newFs(name, path string, m configmap.Mapper) (*Fs, error) {
 
 	// Create a new authorized Drive client.
 	f.client = oAuthClient
-	f.svc, err = drive.New(f.client)
+
+	f.svc, err = drive.NewService(ctx, option.WithHTTPClient(f.client))
 	if err != nil {
 		return nil, errors.Wrap(err, "couldn't create Drive client")
 	}
 
+	f.activitySvc, err = driveactivity.NewService(ctx, option.WithHTTPClient(f.client))
+	if err != nil {
+		return nil, errors.Wrap(err, "couldn't create Drive Activity client")
+	}
+
 	if f.opt.V2DownloadMinSize >= 0 {
-		f.v2Svc, err = drive_v2.New(f.client)
+		f.v2Svc, err = drive_v2.NewService(ctx, option.WithHTTPClient(f.client))
```

Comment:
What is this change about? Is it the New recommended API now?

Reply:
Yep. https://godoc.org/google.golang.org/api/drive/v3#New